src/Settings/index.js

```diff
@@ -1,13 +1,14 @@
 const settings = {}
-let subscribers = []
+let subscribers = {}
```

Comment:
We can change this to a `const` now, because we're not reassigning anymore in `clearSubscribers()`.